doc/changelog.txt

```diff
@@ -35,6 +35,7 @@ Lua:
 UI:
  - Add FPSClampPos config for whether the engine should verify the camera doesn't hit ground/go too far
    this and other FPS camera configs can be changed midgame and will affect the next frame drawn.
+ - Add OverheadMaxHeightFactor config - a float multiplier for maximum overhead camera height.
```

Comment:
FYI: for documentation of config values  .description() exists. i.e.

https://springrts.com/wiki/Springsettings.cfg#3DTrees
https://github.com/spring/spring/blob/403f6cb157da37bbb820611bae40b09f544f1126/rts/Rendering/Env/ITreeDrawer.cpp#L26